python/Makefile

```diff
@@ -49,10 +49,26 @@ style:
 	isort --apply --recursive $(STYLE_TARGETS) --skip-glob "$(EXCLUDE_TARGETS)/*"
 	autoflake -i --remove-all-unused-imports -r $(STYLE_TARGETS) --exclude "$(EXCLUDE_TARGETS)"
 	flake8
+	pyright --version
```

Comment:
Done in b70acaf

Reply:
that, or my github is acting up

Reply:
I am not aware of any reverting, the second commit is actually older, so it is not possible to do that (maybe vice-versa). What part of code is it affecting?

BTW I also have some issues with GH - links on comments are not working fine for me in this PR (maybe because of so many comments)

Reply:
i'm seeing the bad version on github, but you are right, the other commit is older, and my local checkout looks ok